test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -329,3 +329,50 @@ Describe "Import-Module -Force behaviour" -Tag "CI" {
         { Test-Two } | Should -Throw -ErrorId 'CommandNotFoundException'
     }
 }
+
+Describe "Get-Module -ListAvailable behaviour" -Tag "CI" {
```

Comment:
Should these go in `Get-Module.Tests.ps1` in the same directory?

Reply:
Yes, good catch. I have moved it to `Get-Module.Tests.ps1`.